server/src/main/java/io/deephaven/server/table/ops/TableServiceGrpcImpl.java

```diff
@@ -309,6 +312,12 @@ public void fetchPandasTable(FetchPandasTableRequest request,
     public void batch(final BatchTableRequest request,
             final StreamObserver<ExportedTableCreationResponse> responseObserver) {
         GrpcUtil.rpcWrapper(log, responseObserver, () -> {
+            GrpcErrorHelper.checkRepeatedFieldNonEmpty(request, BatchTableRequest.OPS_FIELD_NUMBER);
```

Comment:
I wonder if this fixed Carson's issue?

Reply:
I couldn't find an explicit issue you were referencing. Maybe thought of https://github.com/deephaven/deephaven-core/issues/2721, but I don't think that's relevant.